app/javascript/components/ansible-playbook-edit-catalog-form/schema.js

```diff
@@ -0,0 +1,141 @@
+import { componentTypes } from '@@ddf';
+
+function transformOptions(options) {
```

Comment:
lets use arrow functions
`const transformOptions = (options) => {`

Reply:
No more using this function